### PR TITLE
Feature/form html exercise

### PR DESCRIPTION
**Pull request by EstefaSora125**
Exercise form by Estefania Sora

### CODE REVIEW

File: 4-forms-validations/form-html.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <form>
+        <label for="first-name">
+            First Name: <input type="text" id="first-name" name="user_name" required>
+             <abbr title="required" aria-label="required" >*</abbr>
+         </label>
+         <label for="first-name">
```
Comment (pushdev-code) [inline 4-forms-validations/form-html.html]:
```suggestion
         <label for="Lastname">
```

File: 4-forms-validations/form-html.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <label for="status-5">5</label>
+    </fieldset>
+    <fieldset>
+        <legend>Choose your hobbies</legend>
+        <input type="checkbox" id="taste_1" name="taste_read" value="read">
+        <label for="taste_1">Read books</label>      
+        <input type="checkbox" id="taste_2" name="taste_soccer" value="soccer">
+        <label for="taste_2">Play soccer</label>      
+        <input type="checkbox" id="taste_2" name="taste_origami" value="origami">
+        <label for="taste_2">Do origami</label>
```
Comment (pushdev-code) [inline 4-forms-validations/form-html.html]:
```suggestion
        <input type="checkbox" id="taste_3" name="taste_origami" value="origami">
        <label for="taste_3">Do origami</label>
```

File: 4-forms-validations/form-html.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    </fieldset> 
+    <label for="myfile">Select files:</label>
+        <input type="file" id="myfile" name="myfile" accept="image/png, image/jpeg" required>
+        <abbr title="required" aria-label="required">*</abbr>
+          <button type="submit">Sumbit</button>
```
Comment (pushdev-code) [inline 4-forms-validations/form-html.html]:
```suggestion
          <button>Submit</button>
```

**Review by pushdev-code — CHANGES_REQUESTED** [review]
Just some small comments. If possible try to complete the exercise with your own elements. Mastery Is About How You Practice.

File: 4-forms-validations/form-html.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            Email: <input type="email" id="email" name="user_email" required>
+            <abbr title="required" aria-label="required">*</abbr>
+        </label>
+         <label for="birth-date">
+            Date of birth: <input type="date" id="birth-date" name="user_birth" required>
+            <abbr title="required" aria-label="required">*</abbr>
+         </label>
+         <fieldset>
+            <legend>Gender</legend>
+            <select id="gender_user">
```
Comment (pushdev-code) [inline 4-forms-validations/form-html.html]:
```suggestion
            <legend>Gender</legend>
            <label for="gender">Gender:</label>
            <select id="gender_user">
```

**Review by pushdev-code — CHANGES_REQUESTED** [review]
It's not needed to use `aria-label="required"` in your `<abbr>` you can do: `<input aria-required="true" ....>` In all your inputs.

**Review by juancho11gm — APPROVED** [review]

**Pull request merged**